webapp/portlet/src/main/webapp/vue-apps/popover/main.js

```diff
@@ -23,19 +23,31 @@ Vue.directive('identity-popover', (el, binding) => {
   }
 
   el.addEventListener('mouseover', () => {
-    const rect = el.getBoundingClientRect();
-    document.dispatchEvent(new CustomEvent('popover-identity-display', {
-      detail: Object.assign({
-        offsetX: rect.left + window.scrollX,
-        offsetY: rect.top > 150 + rect.height ? rect.top : rect.bottom + window.scrollY,
-        top: rect.top > 150 + rect.height ? true : false, 
-        identityType: isUser ? 'User' : 'Space',
-        element: el,
-      }, identity || {})
-    }));
+    showPopover(el, identity, isUser);
+  });
+  el.addEventListener('keyup', (event) => {
+    if (event.key === 'Tab') {
+      showPopover(el, identity, isUser);
+    }
+  });
```

Comment:
As I understand, when I type "Tab", all popovers will be displayed in same time, rigth ? 
It is not the expected.

We want to make popover part of the tabulation navigation.

Reply:
the event listener is set on the element not on all the documents so it appears when the TAB on the element 
and closes when leaving the element with "focusout" event

Reply:
Can we simply use events onfocus and onblur ?

I think theses standard event are preferable to a hardcoded key press